metafacture-biblio/src/test/java/org/metafacture/biblio/marc21/Marc21EncoderTest.java

```diff
@@ -114,4 +114,26 @@ public void issue278ShouldNotFailWhenProcessingLeaderEntity() {
         verify(receiver).process(any(String.class));
     }
 
+    @Test
+    public void issue454ShouldNotFailWhenProcessingEntityLeaderAsOneString() {
+        marc21Encoder.startRecord("");
+        marc21Encoder.startEntity(LEADER_ENTITY);
+        marc21Encoder.literal(LEADER_ENTITY, "02602pam a2200529 c 4500");
+        marc21Encoder.endEntity();
+        marc21Encoder.endEntity();
+        marc21Encoder.endRecord();
+
+        verify(receiver).process(matches("00026pam a2200025 c 4500\u001e\u001d"));
+    }
+
+    @Test
+    public void issue454ShouldNotFailWhenProcessingLeaderAsOneString() {
+        marc21Encoder.startRecord("");
+        marc21Encoder.literal(LEADER_ENTITY, "02602pam a2200529 c 4500");
+        marc21Encoder.endEntity();
+        marc21Encoder.endRecord();
+
+        verify(receiver).process(matches("00026pam a2200025 c 4500\u001e\u001d"));
+    }
```

Comment:
Should there be some test for #524 as well?

Reply:
Both `issue454*` tests did so implicitly, but I agree to have an explicit named test.
Added in eb38a1bd29cb7a306561248da428dd60f3c2eaae.